app/views/test_model_view.py

```diff
@@ -0,0 +1,90 @@
+from django.contrib.auth.models import User
+from django.test import Client, TestCase
+from django.utils.http import urlencode
+
+from app.models import Donor
+
+
+class DonorViewTestCase(TestCase):
+    def setUp(self):
+        self.client = Client()
+        self.url_path = "/api/donor"
+
+        user_password = "testing"
+        self.user = User.objects.create_superuser(
+            "tester", email="tester@example.com", password=user_password
+        )
+        self.client.login(username=self.user.username, password=user_password)
+
+        self.test_donor = Donor.objects.create(donor_name="Test Donor")
```

Comment:
What do you think about creating importable generic mock for basic models?

Reply:
Could you say more about your suggestion? I'm not clear on how we could use a generic mock to verify changes made by requests to routes 🤔

Reply:
I was thinking we could have mock definitions that can be used in situations where the model isn't the main test subject but I think it's fine in these scenarios. Ie) `Donation.object.create(donor=createMockDonor(),...)`